### ISSUE TITLE
Enhancement request - Brightness setting for day and night

### ISSUE THREAD

**Issue by jedd** [simgunz/redshift-plasmoid#15]
Fantastic widget, thanks.

It looks like the brightness value has an effect during day and night periods equally.  Would it be feasible to have the brightness value only take effect (via the smooth transition) with the shift to the Night colour temperature?   Alternatively have a separate brightness value associated with the Day colour temperature range?

**Comment by silverhook** [comment]
What about instead of relying on Redshift to change the brightness, simply use the the BlueDevil (or Battery Monitor) DBUS calls?

**Comment by simgunz** [comment]
The aim of that setting is to control the brightness parameter of redshift, but actually there is no need to use it at all. Why one should control brightness trough redshift when I can do it trough the battery plasmoid?

I should remove it completely, but some likes it.

**Comment by silverhook** [comment]
IMHO it might make sense to embed Redshift into the Battery Plasmoid :smile_cat: 

On a more serious note, is it actually possible to automatically adjust the brightness according to time inside the Battery Plasmoid?

**Comment by xlegs** [comment]
What about desktop users? I don't think the Battery Plasmoid works in that
case.

Stefan Zecevic
The little commuter in the big red scooter

**Comment by silverhook** [comment]
@xlegs, good point. But then again, the “Battery Plasmoid” seems to be responsible for screen (and keyboard back-light) brightness even on desktops.

**Comment by jedd** [comment]
Yes, I use a desktop and laptop, and it's convenient having a single icon / control to flip back to 'raw'  screen output in terms of hue and contrast -- for example when working on graphic or photo manipulation after sunset.

**Comment by simgunz** [comment]
So what are the proposed changes here? To control the brightness with dbus via redshift, because even tough it can be controlled via battery widget you would lke to have a one-click control to switch between two level of brightness (together with the change of gamma). RIght?

A note: For plasma 5 there is a new applet to control redshift, developed by someone else. So you may need to suggest this also there:
http://kde-apps.org/content/show.php/Redshift+Control?content=170746
